spec/models/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of :first_name }
+    it { should validate_presence_of :last_name }
```

Comment:
краще такий синтаксис
```suggestion
    it { is_expected.to validate_presence_of :first_name }
    it { is_expected.to validate_presence_of :last_name }
```